.cargo/config.toml

```diff
@@ -1,24 +1,17 @@
 [target.xtensa-esp32-espidf]
 linker = "ldproxy"
 runner = "espflash --monitor"
-rustflags = ["-C", "default-linker-libraries", "-Clink-args=-Wl,-Map=./esp-build.map", "--cfg", "espidf_time32"]
-
+rustflags = ["--cfg", "espidf_time32"]
 
 [target.xtensa-esp32s2-espidf]
 linker = "ldproxy"
 runner = "espflash --monitor"
-rustflags = ["-C", "default-linker-libraries", "--cfg", "espidf_time32"]
+rustflags = ["--cfg", "espidf_time32"]
 
 [target.xtensa-esp32s3-espidf]
 linker = "ldproxy"
 runner = "espflash --monitor"
-rustflags = ["-C", "default-linker-libraries", "--cfg", "espidf_time32"]
-
-[target.riscv32imc-esp-espidf]
```

Comment:
We don't support riscv right now, so it doesn't make sense to me to carry around the block for configuring it.